lib/pbench/server/indexing_tarballs.py

```diff
@@ -267,6 +272,32 @@ def process_tb(self, tarballs):
         else:
             idxctx.set_tracking_id(tracking_id)
 
+        return res.value
+
+    def process_tb(self, tarballs: List[TarballData]) -> int:
+        """Process Tarballs For Indexing and creates report
+
+        Args:
+            tarballs:   List of tarball information tuples
+
+        Returns:
+            status code
+        """
+        res = 0
+        idxctx = self.idxctx
+        error_code = self.error_code
+
+        # Store the list of tarballs as a deque
+        tb_deque = deque(tarballs)
```

Comment:
I'm gonna go out on a limb here and say that that comment does nothing to illuminate the line of code below it.  🤣 

Now if you want to explain what a `deque` is, that might be worthwhile; otherwise, I would just omit the line....